tests/test_worker_pool.py

```diff
@@ -43,9 +43,10 @@ def test_check_workers(self):
         self.assertEqual(len(pool.worker_dict.keys()), 2)
 
         worker_data = list(pool.worker_dict.values())[0]
+        sleep(0.5)
         _send_shutdown_command(worker_data.name, self.connection.connection_pool.connection_kwargs.copy(), delay=0)
         # 1 worker should be dead since we sent a shutdown command
-        sleep(0.2)
+        sleep(0.75)
```

Comment:
I increased the delay time because it takes a bit more time to start and stop the worker with a scheduler